cmd/mysterium_client/interactive/io.go

```diff
@@ -0,0 +1,30 @@
+package interactive
+
+import (
+	"fmt"
+)
+
+const statusColor = "\033[33m"
+const warningColor = "\033[31m"
+const successColor = "\033[32m"
+const infoColor = "\033[93m"
+
+func status(label string, items ...interface{}) {
+	fmt.Printf(statusColor+"[%s] \033[0m", label)
+	fmt.Println(items...)
+}
+
+func warn(items ...interface{}) {
+	fmt.Printf(warningColor + "[WARNING] \033[0m")
+	fmt.Println(items...)
+}
+
+func success(items ...interface{}) {
+	fmt.Printf(successColor + "[SUCCESS] \033[0m")
+	fmt.Println(items...)
+}
+
+func info(items ...interface{}) {
+	fmt.Printf(infoColor + "[INFO] \033[0m")
+	fmt.Println(items...)
+}
```

Comment:
There is a lot of boiler-plate duplication in these methods, maybe it's worth to extract common message pattern?

This seems duplicated, can we reuse `status` function?
```go

func log(statusColor, label string, items ...interfaceP{) {
   fmt.Printf("%s[%s] \033[0m", statusColor, label)
   fmt.Println(items...)
}

func warn(items ...interface{}) {
    log(warningColor, "WARNING", items)
}
func success(items ...interface{}) {
   ....
}

...
```